app/views/templates/_form.html.erb

```diff
@@ -35,6 +35,11 @@
                                                                              "http://www.theforeman.org/manuals/#{SETTINGS[:version].short}/index.html#4.4.3ProvisioningTemplates", :rel => 'external')).html_safe) %>
       <% end -%>
 
+      <% if pxe_with_building_hosts?(@template) -%>
+        <%= alert :class => 'alert-warning', :header => '',
+                  :text => icon_text("warning-sign", (_("Warning: The template has associated at least one host in build mode, to apply the change, trigger building mode again."))) %>
```

Comment:
`Warning: The template is associated to at least one host in build mode. To apply the change, disable and enable build mode on hosts to update the live templates.`